test/jdk/java/util/Locale/LSRDataTest.java

```diff
@@ -56,7 +56,7 @@
     // relative path to the file in the make folder, considering
     // test and make will always exist in the same jdk layout
```

Comment:
The comment refers to the "make" folder. (line 55 as well).

Reply:
Fixed. Thank you for noticing!